### PR TITLE
Jade

### PR DESCRIPTION
**Pull request by JadeEmm**
Please review updates to the pages for pricing, become an agent,  contact us and home

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -64,7 +64,29 @@ export default function Home() {
         {/* Browse Items */}
         <BrowseItems />
         
+        {/* Final Call to Action */}
+<section className="py-8 sm:py-16 text-center">
+<div className="bg-primary text-primary-foreground px-4 py-8 inline-block rounded-lg">
+    <Link className="font-bold text-lg sm:text-2xl" href="/signup">Connect with an Agent</Link>
```
Comment (marmikpatel19) [inline app/page.tsx:70]: is /signup a real page? I don't remember up making a page. we should link directly to the auth flow

File: app/page.tsx [hunk 2]
```diff
@@ -64,7 +64,29 @@ export default function Home() {
...
+</div>
+  <h3 className="text-xl sm:text-2xl mb-4">Create an Account or Sign In</h3>
+</section>
+
         {/* Key Metrics Section */}
+
+        <section className="py-8 sm:py-1 bg-blue-800 text-white">
+  <div className="max-w-md mx-auto text-center">
+    <div className="p-4 bg-blue-800 text-white rounded-lg hover:scale-105 transform transition-all duration-300">
+      <h2 className="text-3xl font-bold mb-0">Join the Agent Success Story</h2>
```
Comment (marmikpatel19) [inline app/page.tsx:80]: is this meant to be a button? and shouldn't our marketing site tailor to seekers and not agents? we can have a less public page for agents.

File: components/BecomeAnAgent.tsx
Comment (marmikpatel19) [inline components/BecomeAnAgent.tsx:1]: nice work!

File: components/ContactUs.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                <i className="fa fa-paper-plane"></i><span className="send-text">SEND</span>
+              </div>
+            </button>
+          </form>
+
+          {/* Right contact page */}
+          <div className="direct-contact-container">
+            <ul className="contact-list">
+              <li className="list-item"><i className="fa fa-map-marker fa-2x"><span className="contact-text place">San Francisco, California</span></i></li>
+              <li className="list-item"><i className="fa fa-phone fa-2x"><span className="contact-text phone"><a href="tel:XX" title="Give me a call">(XXX) XXX-XXXX</a></span></i></li>
```
Comment (marmikpatel19) [inline components/ContactUs.tsx:50]: let's leave out the placeholders for now. we can add them in as needed

File: components/ContactUs.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+              <div className="col-sm-12">
+                <input type="email" className="form-control" id="email" placeholder="EMAIL" name="email" required />
+              </div>
+            </div>
+
+            <textarea className="form-control" rows={10} placeholder="MESSAGE" name="message" required></textarea>
+
+            <button className="btn btn-primary send-button" id="submit" type="submit">
+              <div className="alt-send-button">
+                <i className="fa fa-paper-plane"></i><span className="send-text">SEND</span>
```
Comment (marmikpatel19) [inline components/ContactUs.tsx:41]: just UI? the contact form doesn't link anywhere

can you make it functional? let's keep it simple. let's link to a google spreadsheet for now or even simpler just send us an email 

even simpler would be to just remove contact form and just put our email. we can add in contact form later.

File: components/KeyMetrics.tsx [hunk 5]
```diff
@@ -7,24 +7,28 @@ interface KeyMetricProps {
...
 };
 
 const KeyMetrics: React.FC = () => {
   const metrics = [
-    { value: "500+", caption: "Hired Agents" },
-    { value: "10,000+", caption: "Applications Submitted" },
-    { value: "+30%", caption: "Application to Interview Conversion Success Rate" },
+    { value: "1+", caption: "Hired Agents" },
+    { value: "50+", caption: "Applications Submitted" },
+    { value: "1st", caption: "Be One of the First Submissions With Fast Track" },
```
Comment (marmikpatel19) [inline components/KeyMetrics.tsx:25]: left some comments on discords about fast track

**Review by marmikpatel19 — CHANGES_REQUESTED** [review]
lots of great work!

In the future, please make a branch & pr for each feature instead of doing everything on one branch and merging in one pr. This way we have modularity and others can keep up or take over work. Also, please make your PR names descriptive (e.g., "contact us page")

Left some feedback for you to review. also, you have some merge conflicts that you need to resolve before merging

File: app/page.tsx (hunk 2 above)
Comment (JadeEmm) [reply app/page.tsx:80]: No, not meant to be a button. Yes, 'Join the Agent Success Story' is targeted towards seekers, as in they will find career/job success by using Agent and be a success story

**Pull request closed without merging**